app/services/cache_service.rb

```diff
@@ -43,7 +50,21 @@ def listing_unchanged?(prev_cached_listing, fresh_listing)
       (prev_cached_listing['LastModifiedDate'] == fresh_listing['LastModifiedDate'])
   end
 
+  def listing_images_unchanged?(prev_cached_listing, _fresh_listing)
+    prev_cached_listing_images = prev_cached_listing&.dig('Listing_Images')
+    fresh_listing_images = fresh_listing_images&.dig('Listing_Images')
```

Comment:
This line used to be 
```ruby
fresh_listing_images = fresh_listing.present? && fresh_listing['Listing_Images']
```

What happened to using the `fresh_listing` argument? Now, it looks like `fresh_listing_images&.dig` will always be `nil` because it's undefined right?